celery/backends/database/models.py

```diff
@@ -26,7 +25,7 @@ class Task(ResultModelBase):
                    primary_key=True, autoincrement=True)
     task_id = sa.Column(sa.String(155), unique=True)
     status = sa.Column(sa.String(50), default=states.PENDING)
-    result = sa.Column(PickleType, nullable=True)
+    result = sa.Column(sa.BLOB, nullable=True)
```

Comment:
Can we use `LargeBinary` with custom length instead so that the largest possible datatype is used (at least for MySQL):
- https://stackoverflow.com/a/43791881/920374
- https://dev.mysql.com/doc/refman/5.7/en/storage-requirements.html#data-types-storage-reqs-strings

I think PostgreSQL `BYTEA` columns have a limit of 1GB, so that RDBMS will be covered too.

Reply:
Sure, will update it